docs/ens_overview.rst

```diff
@@ -11,11 +11,15 @@ The :mod:`ens` module is included with web3.py. It provides an interface to look
 domains and addresses, add resolver records, or get and set metadata.
 
 
-.. note:: ENSIP-15 introduced a new standard for ENS name normalization. This standard
-    is implemented in web3.py via the ``ensip15_normalization`` flag on the ``ENS`` /
-    ``AsyncENS`` class. It is also available as a flag on relevant ``ens.utils.py``
-    utility methods. For more information, refer to the :ref:`ensip15_normalization`
-    section.
+.. note::
+
+    web3.py ``v6.1.0`` introduced ENS name normalization standard
+    `ENSIP-15 <https://docs.ens.domains/ens-improvement-proposals/ensip-15-normalization-standard>`_.
+    This update to ENS name validation and normalization won't affect ~99%
```

Comment:
Could add a statement about the prior "art" of normalization in the ENS module. Not sure if it would help to call out specific cases that prompted the breaking change but that might be useful too.

Reply:
> Could add a statement about the prior "art" of normalization in the ENS module.

Any suggestion here? I'm not sure I understand.

> Not sure if it would help to call out specific cases that prompted the breaking change but that might be useful too.

Not sure here either. I think the biggest driver was that there wasn't a properly defined standard to begin with, or at least clear lines drawn when it comes to complex names. I lean towards deferring to the proper ENS documentation for answering the why questions but if you think of a clarifying addition I'm happy to pop it in here 👀 .

Reply:
Ah, heard. Yeah I think that can be addressed in the proper ENS docs as long as we link out to it here. There's a motivation section in the ``ENSIP-15`` documentation that I think addresses this pretty well.